app/Http/Controllers/TaxRateController.php

```
@@ -0,0 +1,137 @@
+            'percentage_basis_points' => ['required', 'integer', 'min:0', 'max:1000000'],
+            'country' => ['nullable', 'string', 'max:2'],
+            'active' => ['boolean'],
+            'valid_from' => ['nullable', 'date'],
+            'valid_until' => ['nullable', 'date', 'after_or_equal:valid_from'],
+            'description' => ['nullable', 'string', 'max:1000'],
+        ]);
+
+        $validated['user_id'] = auth()->id();
+        $validated['active'] = $request->has('active');
+
+        $taxRate = TaxRate::create($validated);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$taxRate'. (undefined)

(UnusedLocalVariable)